.ci/pitest.sh

```diff
@@ -135,10 +135,11 @@ pitest-ant)
 pitest-indentation)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
   declare -a ignoredItems=(
+  "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>                &#38;&#38; tree.getColumnNo() &#60; realStart.getColumnNo()) {</span></pre></td></tr>"
   "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>            if (colNum == null || thisLineColumn &#60; colNum) {</span></pre></td></tr>"
-  "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>        if (currLine &#60; realStart) {</span></pre></td></tr>"
   "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>            if (toTest.getColumnNo() &#60; first.getColumnNo()) {</span></pre></td></tr>"
   "ArrayInitHandler.java.html:<td class='covered'><pre><span  class='survived'>        if (firstChildPos &#62;= 0) {</span></pre></td></tr>"
+  "ArrayInitHandler.java.html:<td class='covered'><pre><span  class='survived'>        return true;</span></pre></td></tr>"
```

Comment:
I had to add this line to resolve https://github.com/checkstyle/checkstyle/pull/8038#discussion_r415086516

Problem is, it is logging such nesting errors only during `ant-phase-verify` but if the same file is tested against the same `checkstyle_config` with generated `snapshot-jar` then those errors are not logged.

Infact, Any type of array nesting has no effect for test cases inside the `indentationcheckTest.java`, So it get caught by pitest.